tests/templates/kuttl/tls/check-tls.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+import trino
+import argparse
+import sys
+import yaml
+
+
+if not sys.warnoptions:
+    import warnings
+warnings.simplefilter("ignore")
```

Comment:
```suggestion
if not sys.warnoptions:
    import warnings
    warnings.simplefilter("ignore")
```

Reply:
What warnings are you trying to silence anyway? Why aren't they fixed instead?

Reply:
I checked without the warnings and nothing shows up anymore -> removed